react/services/getBulkImportList.ts

```diff
@@ -13,13 +13,19 @@ const getBulkImportList = (account: string) =>
     .then(v => (v.data as unknown) as ImportDetails[])
     .then(bulkImports =>
       bulkImports
-        .filter(item => item.status !== 'ReadyToImport')
+        .filter(
```

Comment:
I hadn't noticed in the other PR, but it would be better to replace this promise syntax here with async/await.